cmd/webhook/main.go

```diff
@@ -19,6 +19,10 @@ package main
 import (
 	"context"
 
+	servingv1alpha1 "knative.dev/serving/pkg/apis/serving/v1alpha1"
```

Comment:
domainmapping v1alpha1 is deprecated https://github.com/knative/serving/issues/14059 and will be removed after v1.11 cut https://github.com/knative/serving/issues/14060. 
We should add v1alpha1 if this PR will be merged before v1.11 cut but it will be dropped very soon :sweat_smile: